sql/src/main/java/io/druid/sql/calcite/aggregation/ApproxCountDistinctSqlAggregator.java

```diff
@@ -91,7 +91,7 @@ public Aggregation toDruidAggregation(
     final AggregatorFactory aggregatorFactory;
 
     if (input.isDirectColumnAccess() && rowSignature.getColumnType(input.getDirectColumn()) == ValueType.COMPLEX) {
-      aggregatorFactory = new HyperUniquesAggregatorFactory(name, input.getDirectColumn());
+      aggregatorFactory = new HyperUniquesAggregatorFactory(name, input.getDirectColumn(), false, true);
```

Comment:
Why rounding?

Reply:
This is for the SQL layer; the `COUNT DISTINCT` function in SQL is int-typed, and surprising results can happen from expressions involving counts if it's not treated as an int.